LANE_server/settings.py

```diff
@@ -31,13 +31,11 @@
     '127.0.0.1',
     '.herokuapp.com',
     '.up.railway.app',
+    '.lanl.gov',
 ]
 
 # For FE and BE hosted locally on different ports
-CORS_ALLOWED_ORIGINS = [
-    "http://localhost:3000",
-    "http://ec2-44-202-29-2.compute-1.amazonaws.com",
-]
+CORS_ALLOWED_ORIGINS = ['*']
```

Comment:
Is this safe?

Reply:
From what I am reading, it is relatively safe minus a few edge cases. Still, it's not exactly clear cut so I will be just reverting to the hard coded list